test/formats/test_vorbis.py

```diff
@@ -208,6 +208,7 @@ class FLACTest(CommonVorbisTests.VorbisTestCase):
         '~channels': '2',
         '~sample_rate': '44100',
         '~format': 'FLAC',
+        '~filesize':'6546'
```

Comment:
Can you please add this check for other formats as well? As it is the responsibility of each format to call the `_info` method (and in some cases might not do so) this makes sure this works for all formats and does not break.

Which reminds me that getting the `~filesize` should also be added to https://github.com/metabrainz/picard/blob/master/picard/formats/wav.py#L237 . That's the legacy WAVE file support which does not use mutagen. While me might remove this at some point we should still set the filesize as long as we have the code still.

And minor coding style issue, but there should be a space after the colon.

Reply:
Sure, I will soon add this after probably 15 Jan (due to ongoing semester exams :( )